consensus/module.go

```diff
@@ -65,47 +64,53 @@ type ConsensusModule struct {
 	MaxBlockBytes uint64
 }
 
-func Create(configPath, genesisPath string, useRandomPK bool) (modules.ConsensusModule, error) {
-	cm := new(ConsensusModule)
-	c, err := cm.InitConfig(configPath)
-	if err != nil {
-		return nil, err
+func Create(runtime modules.Runtime) (modules.Module, error) {
+	var m ConsensusModule
+	return m.Create(runtime)
+}
+
+func (*ConsensusModule) Create(runtime modules.Runtime) (modules.Module, error) {
+	var m *ConsensusModule
```

Comment:
It doesn't make sense to me that you have a receiver variable that is ignored, but then the line below you declare another consensus module `var m *ConsensusModule`

Reply:
🤦‍♂️ you are totally right

https://github.com/pokt-network/pocket/pull/235/files#diff-eba95a8b904f8186683d2f9c106a3dc021f014918420e6a074d0a2b4e59c261dR66-R68